project/Dependencies.scala

```diff
@@ -75,4 +75,15 @@ object Dependencies {
     m.excludeAll(ExclusionRule(organization = "com.google.code.findbugs", name = "jsr305"))
   )
 
+  lazy val migrationDeps = Seq(
+    "org.mongodb.scala" %% "mongo-scala-driver" % "2.5.0",
+    "org.scalikejdbc"   %% "scalikejdbc"        % "3.3.0",
+    "org.scalaz"        %% "scalaz-zio"         % "0.5.1",
```

Comment:
NB: despite the name, this is a zero-dependency library that does not inject the whole scalaz ecosystem (as opposed to cats-effect).